test/fakeintake/app/main.go

```diff
@@ -18,17 +19,33 @@ func main() {
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, os.Interrupt, syscall.SIGTERM)
 
-	fmt.Println("⌛️ Starting fake intake")
-	fi := fakeintake.NewServer(8080)
-	fmt.Println("🏃 Fake intake running")
+	log.Println("⌛️ Starting fake intake")
+	ready := make(chan bool, 1)
+	fi := fakeintake.NewServer(fakeintake.WithPort(8080), fakeintake.WithReadyChannel(ready))
+	fi.Start()
+	timeout := time.NewTimer(250 * time.Millisecond)
```

Comment:
The value is a bit aggressive for constrained environments, I'd go more around 1-5 seconds.

Reply:
Increased to 5 seconds